.github/actions/has-changesets/action.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Has Changesets'
+description: 'Checks if `.changeset` directory has valid changesets'
+outputs:
+  has-changesets:
+    description: "Whether `.changeset` directory has at least one changeset"
+    value: ${{ steps.check-changesets.outputs.has-changesets }}
+runs:
+  using: "composite"
+  steps:
+    - name: Check .changeseet folder for changesets
+      id: check-changesets
+      shell: bash
+      run: |
+        if (find .changeset -type f -name '*.md' ! -name 'README.md' | grep -q .) ; then
```

Comment:
This line checks whether `.changeset` folder has any markdown files other than the README.